components/map/basic/src/leaflet/layer-manager.js

```diff
@@ -18,8 +26,7 @@ export default class LayerManager {
         {
           app_code: appCode,
           app_id: appId,
-          attribution:
-            'Map &copy; 1987-2017 <a href="http://developer.here.com">HERE</a>',
+          attribution: attribution,
```

Comment:
In this case, you could use the shorthand for assigning keys with the same value name.